app/views/admin/welcome/_roasts.html.erb

```diff
@@ -0,0 +1,49 @@
+<div class="row">
+  <div class="panel panel-default">
+    <div class="panel-body">
+      <h2>Top 10 Roasts</h2>
+      <table class="table table-hover" style="width: 100%">
+        <colgroup>
+          <col span="1" style="width: 3%;" />
+          <col span="1" style="width: 32%;" />
+          <col span="1" style="width: <%= col_width %>%;" />
+          <col span="1" style="width: <%= col_width %>%;" />
+          <col span="1" style="width: <%= col_width %>%;" />
+          <col span="1" style="width: <%= col_width %>%;" />
+          <% if params[:controller] == "admin/welcome" %>
+          <col span="2" style="width: 5%;" />
+          <% end %>
+        </colgroup>
+        <tr>
+          <th></th>
+          <th>Roast</th>
+          <th>Upvotes <span class="glyphicon glyphicon-thumbs-up"></span></th>
+          <th>Downvotes <span class="glyphicon glyphicon-thumbs-down"></span></th>
+          <th>Score</th>
+          <th>Total Votes (Count)</th>
+          <% if params[:controller] == "admin/welcome" %>
+          <th colspan="2"></th>
+          <% end %>
+        </tr>
+        <ol>
+          <% report.each_with_index do |(roast, stats), index| %>
+          <tr>
+            <td>
+              <%= "##{index+1}" %>
+            </td>
+            <td><%= roast.company %> <%= roast.name %></td>
+            <td><%= stats[:good] %></td>
+            <td><%= stats[:bad] %></td>
+            <td><%= number_to_human stats[:score] %></td>
+            <td><%= stats[:count] %></td>
+            <% if params[:controller] == "admin/welcome" %>
+            <td><%= link_to 'Edit', edit_admin_roast_path(roast) %></td>
+            <td><%= link_to 'Delete', admin_roast_path(roast), method: :delete, data: { confirm: 'Are you sure you want to delete this batch?' } %></td>
+            <% end %>
```

Comment:
This might need to be refactored. There is too much confusion on what is going on. 

We are calling stats[:method] but then have roast object and params[:controller]
Ideally we just want to have objects we send message to like `object.method` see if this is possible. We can have a look at this afternoon together as well